File: iCovieAdmin-Ng/New folder/src/app/core/auth/_models/Masters/businesscategory.model.ts


export class BusinessCategoryModel {
    id: number;    
    businesscategory:string;
    shortname:string;   
    cid:number; 
    isCore: boolean = false;    
    clear(): void {
        this.id= undefined;
        this.businesscategory= '';        
        this.cid = undefined;      
        this.isCore=false;
	}
}

File: iCovieApi/iCovieApi/Models/Master/BusinessCategoryModel.cs

using System;

using System;
using Newtonsoft.Json;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace iCovieApi.Models
{
    public class BusinessCategoryModel
    {
        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
        public int id { get; set; }

        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
        public string businesscategory { get; set; }

        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
        public string shortname { get; set; }


        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
        public int cid { get; set; }
    }
}